Documentation/cmdref/index_cilium-agent.rst

```diff
@@ -0,0 +1,16 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+cilium-agent Command Reference
```

Comment:
Comparing this with the current version, it looks like all CLIs now have the suffix `Command Reference` appended. This looks like too much information, especially in the navigation.

Before

![screenshot-2025-06-13-09-06-18](https://github.com/user-attachments/assets/77f57d11-755e-470d-bf78-b171c1603296)

After

![screenshot-2025-06-13-09-06-02](https://github.com/user-attachments/assets/9da7791b-0d55-41ba-872e-ef9646639077)

Reply:
Updated 👍